firefox-ios/Client/Frontend/Home/Homepage Rebuild/Pocket/PocketManager.swift

```diff
@@ -0,0 +1,21 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+class PocketManager {
+    private let pocketAPI: PocketStoriesProviding
+    private let storyProvider: StoryProvider
+    private var pocketItems: [PocketItem]?
```

Comment:
realize that we don't need this anyone, will remove in future PR so I don't have to re-run bitrise again (took 4x to make green) 🤣 unless there are some other major changes I should make along with this PR.